ubiconfig/_impl/loaders/local.py

```diff
@@ -28,7 +29,11 @@ def load(self, file_name):
             raise
 
         # validate input data
-        validate_config(config_dict)
+        try:
+            validate_config(config_dict)
+        except ValidationError as e:
+            LOG.error("%s FAILED schema validation:\n%s\nSkip for now", file_name, e)
+            return
```

Comment:
To me looks like this should be raise as with YAMLerror.

Reply:
I meant that instead of 'return', we should 'raise' here. As we do with yaml.safe_load.

Reply:
I guess the idea is to not raise errors anymore, just log it. YAMLError should also not raise I guess @JayZ12138.

Reply:
@alexandrevicenzi, yes you are right.